gems/sorbet-runtime/lib/types/types/typed_hash.rb

```diff
@@ -23,7 +23,7 @@ def values
     end
 
     def type
-      @type ||= T::Utils.coerce([@inner_keys, @inner_values])
+      @type ||= T::Utils.coerce([keys, values])
     end
 
     def build_type
```

Comment:
I just want to note that this method definition is no longer needed, since the `super` call will build `type`, which, with the new code structure, will trigger a build of `keys` and `values` anyway.